assignment/scripts/loops-practice.js

```diff
@@ -11,37 +11,71 @@ for (let i=0; i<4; i++) {
 }
 
 // 1.a. TODO: Write a for loop to console.log the numbers from 0 to 5 
-//   - Which part of the example loop do you need to change to do this?
+//   - Which part of the example loop do you need to change to do this? 
+// changed the number "4" to "6"
 console.log('count from 0 to 5');
 
+for (let i=0; i<6; i++) {  
+  console.log(i);
+}
+
+
 // 1.b. TODO: Write a for loop to console.log the numbers from 3 to 5
-//   - Which part of the example loop do you need to change to do this?
+//   - Which part of the example loop do you need to change to do this? 
+// changed the value of "i" from "0" to "3"
+// this starts the count off at 3 and adds one number until it reached "5"
 console.log('count from 3 to 5');
-
+for (let i=3; i<6; i++) {  
+  console.log(i);
+}
 // 1.c. TODO: Write a for loop to console.log EVEN numbers from 2 to 10
 //   - Which part of the example loop do you need to change to do this?
+// changed the value of "i" to "2" so the loop starts at 2, changed the condition to be "i" is less than 11
+// which tells the loop to stop at 10, changed the update expression to add "2", rather than "1" which tells the loop to only show even numbers.
 console.log('count even numbers from 2 to 10 (2, 4, 6, 8, 10):');
-
+for (let i=2; i<11; i+=2) {  
+  console.log(i);
+}
 // 1.d. (STRETCH) TODO: Write a for loop to do a countdown from 5 to 0
 console.log('STRETCH: countdown from 5 to 0');
-
+for (let i=5; i>=0; i-=1) {
+  console.log(i);
+}
 
 // 2. For of loops  
 console.log('---- 2. For Of loop ----');
 let stars = ['Polaris', 'Gacrux', 'Formalhaut', 'Rigel', 'Deneb']
 
 // 2.a. TODO: Write a for of loop to console.log each star in the 'stars' array
 console.log('Some stars:');
+for (let i=0; i<stars.length; i++) {
```

Comment:
Careful! The instructions say to use a for..of loop here, but this is a for-with-semicolons.